.travis.yml

```diff
@@ -21,13 +21,13 @@ branches:
 
 matrix:
   include:
-    - name: "Code Style Check"
-      env: CHECK_CS=1
-    - name: "Unit tests"
-      env: PHPUNIT_CONFIG='phpunit.xml'
-    - name: "Unit tests (PHP 7.4)"
-      env: PHPUNIT_CONFIG='phpunit.xml'
-      php: 7.4
+    #    - name: "Code Style Check"
+    #      env: CHECK_CS=1
+    #    - name: "Unit tests"
+    #      env: PHPUNIT_CONFIG='phpunit.xml'
+    #    - name: "Unit tests (PHP 7.4)"
+    #      env: PHPUNIT_CONFIG='phpunit.xml'
+    #      php: 7.4
     - name: "Admin-UI tests on Clean P"
       env:
         - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
```

Comment:
By the way, apparently `docker-compose` is available (built-in) in GH actions @mnocon. So if you ever would like to transfer behat / selenium tests there as well, there is an option.

Reply:
Yes, we plan to switch the selenium tests to Github Actions as well, but this will be a bigger task and I haven't investigated yet what's the best way to do it.

Good to know that docker-compose is available 👍